src/features/bookings/components/BookingDetailsCancelButton.native.test.tsx

```diff
@@ -102,6 +103,29 @@ describe('<BookingDetailsCancelButton />', () => {
       )
     ).toBeTruthy()
   })
+
+  it('should display button if confirmationDate is null and an expiration date message if isBookingIsDigital', () => {
```

Comment:
should display cancel button and expiration date message when confirmation date is null and that's it a digital booking